src/main/java/org/terasology/gookeeper/system/BreedingSystem.java

```diff
@@ -19,55 +19,69 @@
-import org.terasology.gookeeper.ui.GooeyActivateScreen;
+import org.terasology.gookeeper.component.AggressiveComponent;
+import org.terasology.gookeeper.component.NeutralComponent;
+import org.terasology.gookeeper.component.FriendlyComponent;
+import org.terasology.gookeeper.component.FactorComponent;
+import org.terasology.gookeeper.component.HungerComponent;
+import org.terasology.gookeeper.event.AfterGooeyBreedingEvent;
+import org.terasology.gookeeper.event.BeginBreedingEvent;
+import org.terasology.gookeeper.event.SelectForBreedingEvent;
 import org.terasology.logic.behavior.BehaviorComponent;
 import org.terasology.logic.behavior.asset.BehaviorTree;
-import org.terasology.logic.characters.CharacterHeldItemComponent;
 import org.terasology.logic.characters.CharacterMovementComponent;
-import org.terasology.logic.common.ActivateEvent;
+import org.terasology.logic.characters.StandComponent;
+import org.terasology.logic.characters.WalkComponent;
+import org.terasology.logic.characters.events.OnEnterBlockEvent;
 import org.terasology.logic.common.DisplayNameComponent;
 import org.terasology.logic.delay.DelayManager;
 import org.terasology.logic.delay.DelayedActionTriggeredEvent;
-import org.terasology.logic.delay.PeriodicActionTriggeredEvent;
-import org.terasology.logic.health.DoDestroyEvent;
-import org.terasology.logic.health.EngineDamageTypes;
-import org.terasology.logic.health.HealthComponent;
-import org.terasology.logic.inventory.InventoryManager;
+import org.terasology.logic.location.LocationComponent;
 import org.terasology.logic.players.LocalPlayer;
-import org.terasology.network.ClientComponent;
-import org.terasology.physics.Physics;
+import org.terasology.math.geom.Quat4f;
+import org.terasology.math.geom.Vector3f;
+import org.terasology.math.geom.Vector3i;
+import org.terasology.network.ColorComponent;
+import org.terasology.physics.components.RigidBodyComponent;
 import org.terasology.registry.In;
 import org.terasology.registry.Share;
+import org.terasology.rendering.assets.material.Material;
+import org.terasology.rendering.assets.skeletalmesh.SkeletalMesh;
+import org.terasology.rendering.assets.texture.Texture;
+import org.terasology.rendering.logic.MeshComponent;
+import org.terasology.rendering.logic.SkeletalMeshComponent;
+import org.terasology.rendering.nui.Color;
 import org.terasology.rendering.nui.NUIManager;
-import org.terasology.rendering.nui.layers.ingame.inventory.GetItemTooltip;
-import org.terasology.rendering.nui.widgets.TooltipLine;
 import org.terasology.utilities.Assets;
 import org.terasology.utilities.random.FastRandom;
 import org.terasology.utilities.random.Random;
 import org.terasology.world.BlockEntityRegistry;
-import org.terasology.world.WorldProvider;
-import org.terasology.worldlyTooltipAPI.events.GetTooltipIconEvent;
-import org.terasology.worldlyTooltipAPI.events.GetTooltipNameEvent;
+import sun.text.resources.en.FormatData_en_IE;
+
+import java.math.RoundingMode;
+import java.util.Vector;
```

Comment:
Is this import used?
Terasology uses it's own vectors so I'm unsure as to why this exists